test/cpp/api/autograd.cpp

```diff
@@ -896,9 +896,12 @@ std::tuple<torch::Tensor, torch::Tensor> two_arg_inplace_op(const torch::Tensor&
 
 std::tuple<torch::Tensor, torch::Tensor> two_pairs_of_view_op(const torch::Tensor& self, const torch::Tensor& other) {
   // This is not allowed. We test below that this calling into the boxed kernel will raise an error
-  auto self_view = self.view(-1);
-  auto other_view = other.view(-1);
-  return std::tuple<torch::Tensor, torch::Tensor>(self_view, other_view);
+  return std::tuple<torch::Tensor, torch::Tensor>(self, other);
```

Comment:
I think we are ok because alias info still says that the output aliases the input. We are basically just doing `alias`.
If the function itself returns `.view`, then it'd still have a view relationship even if the op is registered in a composite way, and we wouldn't be testing it properly.